Rules/CommonScripts/HelpfulDeathTips.cfg

```diff
@@ -98,5 +97,21 @@ tips = To hide these tips, and other in-game help, hit F1.;
 		Knights can use their swords to destroy wooden blocks and doors, but not stone;
 		Doors and siege weapons are vulnerable to explosives.;
 		Bison can be tamed with burgers.;
-		Resources can be resupplied by changing classes in workshops, not just at the tent!;
+		You can get resupplies in class workshops, not just at the tent!;
 		You can highlight useful items for your class by holding the pickup key.;
+		As a knight, you can always jab the enemy after successfully slashing them to deal extra heart of damage.;
+		You should always have one transport tunnel near your tent and another one at your tower for quick transportation.;
+		You should always have a working saw trap on your spawn tunnel.;
```

Comment:
This is a specific building strategy which I don't think needs to be covered in the tip.

Reply:
Having one tunnel near your tent and another tunnel (tunnels) somewhere in front tower is the only way of using tunnels that makes sense. It could be changed to something like "if you're using tunnels, make sure they are easily accessible and trapped" so it doesnt outright tell you to build this and build that.